source/material_model/visco_plastic.cc

```diff
@@ -411,6 +418,9 @@ namespace aspect
 
           const double finite_difference_accuracy = 1e-7;
 
+          // A new material model inputs variable that uses the modified strain rate and pressure.
+          MaterialModel::MaterialModelInputs<dim> in_new = in;
```

Comment:
Here you need a new name, but try to give it a name that explains the meaning. E.g. `in_derivatives`